frontend/src/pages/articles/my-articles.tsx

```diff
@@ -11,13 +14,35 @@ import { MOCKED_REACTIONS, MOCKED_TAGS } from './libs/constants.js';
 const MyArticles: React.FC = () => {
   const dispatch = useAppDispatch();
   const { articles } = useAppSelector(({ articles }) => articles);
+  const { hasMore, load } = usePagination();
+
+  const handleLoadArticles = useCallback(() => {
+    void load(async (skip: number, take: number) => {
+      const data = await dispatch(
+        articlesActions.fetchOwn({
+          take,
+          skip,
+        }),
+      ).unwrap();
+
+      return data.items.length > 0;
```

Comment:
```suggestion
      return Boolean(data.items.length);
```

Reply:
Btw, I was wondering what's the point using `Boolean` to make this check, is this just for not using `0` (kind of a magic number)?

Reply:
Yep, just a short form